CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 33.2.2
+
+* Require `config` in JsonClient
```

Comment:
This sounds like a new requirement that clients need to fulfil, which sounds like a breaking change. "Fix JsonClient not explicitly requireing the config" ?